tests/test_browser.py

```diff
@@ -4558,6 +4556,9 @@ def test_emscripten_request_animation_frame(self):
   def test_emscripten_request_animation_frame_loop(self):
     self.btest(path_from_root('tests', 'emscripten_request_animation_frame_loop.c'), '0')
 
+  def test_request_animation_frame(self):
+    self.btest('request_animation_frame.cpp', '0', also_proxied=True)
```

Comment:
Ah, the also_proxied=True probably answers my question.

Reply:
It's a different option, but it's enough to get it running in a worker, which is a good enough test here.

proxy-to-worker basically just runs the entire program in a worker, and does some proxying of mouse and keyboard events, simple 2D canvas display, etc. - enough to run e.g. Doom in a worker. It doesn't use pthreads or anything like that. We may eventually reimplement it using pthreads.